app/services/data_migrations/backfill_hesa_trn_submissions.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module DataMigrations
```

Comment:
If you generate the data migration using this: [module DataMigrations](https://github.com/ilyakatz/data-migrate#generating-migrations)

Then it will create the file with a timestamp similar to normal migration. It'll then run it automatically on deployment to production and create an entry in the `data_migrations` table.

Reply:
ahh for someone reason I looked to check if we were using data-migrate but didn't clock it 😆